src/bp/nlu/typings.ts

```diff
@@ -10,7 +10,7 @@ export interface NLUServerOptions {
 
   verbose: number
   doc: boolean
-  logFilter: string[]
+  logFilter: string[] | undefined // if undefined, all logs are displayed
```

Comment:
I'm just wondering why you aren't using the `process.env.DEBUG` system like everywhere else. The way you set it here means that once the server is started, you cannot enable or disable debugging, you must restart the server, which may means that the situation you are trying to debug is no longer valid.

We could easily make something so we can propagate "setDebugScopes" between the web worker, nlu and studio, so we can enable/disable debug scopes during runtime.

Reply:
This is exactly the case, there is no way to change debug scopes once the server is running.

I think having a shared debug scopes between all these apps is an unhealthy dependency. It is not tech/language agnostic and will end up  being more of a pain point than any help it provides.

Also, I believe the debug scopes data structure is contained in memory so I don't know how such a shared debug scopes system would work between process since there is no way to share memory...

Finally, I kind of hate the fact that there is more than one logger in botpress. There is the main logger with `.debug()` method and the `DEBUG()` objects that both compete with each other. I prefer staying far away from this mess.